src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -206,14 +206,16 @@ Once code is built and you are ready to install it on a robot, you can run
 
 `[options]` may be one, or more, of the following
 
-| Option | Description                                                                    |
-| :----: | :----------------------------------------------------------------------------- |
-|   -u   | The user to install to on the target. Defaults to the user in the Docker image |
-|   -t   | Install toolchain to the target                                                |
-|  -cn   | Only install new config files. This is the default                             |
-|  -cu   | Update config files on the target that are older than the local files          |
-|  -co   | Overwrite all config files on the target                                       |
-|  -ci   | Ignore all changes to config files (installs no config files)                  |
+| Option | Description                                                                                                                                       |
+| :----: | :------------------------------------------------------------------------------------------------------------------------------------------------ |
+|   -u   | The user to install to on the target. Defaults to the user in the Docker image (this default user should almost always be correct for our robots) |
+|   -t   | Install toolchain to the target                                                                                                                   |
+|  -cn   | Only install new config files. This is the default                                                                                                |
+|  -cu   | Update config files on the target that are older than the local files                                                                             |
+|  -co   | Overwrite all config files on the target                                                                                                          |
+|  -ci   | Ignore all changes to config files (installs no config files)                                                                                     |
+
+The toolchain is a collection of 3rd party libraries that have been built into the Docker images. On a new robot, all roles will fail to run unless the toolchain is installed and, more often than not, if you are receiving errors on the robot saying something to the effect of "cannot open shared object file: No such file or directory" this will usually be solved by installing the toolchain.
```

Comment:
Sometimes you have to run `sudo ldconfig` on the robots even after installing the toolchain. The toolchain install is supposed to do it but sometimes it still won't work until you run it again

Reply:
Really? That seems strange